### PR TITLE
fix(network): add network module to status and about

### PR DESCRIPTION
**Pull request by WikiRik**
No new frontend pages yet, just adding the existance of the module in some pages.

### CODE REVIEW

**Comment by WikiRik** [comment]
It is ready for review, but I don't want to merge it until we have network in the setup

File: src/views/static/Status.vue [hunk 1]
```diff
@@ -87,6 +87,14 @@ export default {
           isAlive: 'Waiting...',
           changelog: 'https://github.com/AEGEE/discounts/blob/master/CHANGELOG.md'
         },
+        network: {
+          roundTrip: null,
+          version: '-',
```
Comment (LeonVreling) [inline src/views/static/Status.vue:92]: I assume we still need to put the actual version numbers here

**Review by LeonVreling — APPROVED** [review]

File: src/views/static/Status.vue (hunk 1 above)
Comment (WikiRik) [reply src/views/static/Status.vue:92]: It does some IT magic, but it works

Comment (serge1peshcoff) [reply src/views/static/Status.vue:92]: it's replaced in the function that fetches the Github stuff

**Review by serge1peshcoff — APPROVED** [review]

**Pull request merged**